fix(ins): fix create doc for windows and typos (#3854)
scaleway/docs-content: compute/instances/how-to/create-an-instance.mdx

        - **Choose an Availability Zone**, which represents the geographical region where your Instance will be deployed.
        - **Choose an Instance type**. <br />
          Instance offers vary in pricing, processing power, memory, storage, and bandwidth. [Discover the best Instance type for your needs](/compute/instances/reference-content/choosing-instance-type/).
        - **Choose an image** to run on your Instance. <br />
          This can be an operating system, an InstantApp, or a custom image. [Check all available Linux distributions and InstantApps](/compute/instances/reference-content/images-and-instantapps/).
        - **Name your Instance**, or leave the randomly-generated name in place. Optionally, you can add [tags](/compute/instances/concepts/#tags) to help you organize your Instance.

>         - **Add volumes**, which are storage spaces used by your Instances. A block volume with a default name and 5,000 IOPS is automatically provided for your system volume. You can customize this volume and attach up up to 16 local and/or block type volumes as needed.

Reviewer: ```suggestion
        - **Add volumes**, which are storage spaces used by your Instances. A block volume with a default name and 5,000 IOPS is automatically provided for your system volume. You can customize this volume and attach up to 16 local and/or block type volumes as needed.
```

    2. Click **Create Instance**. The [Instance creation page](https://console.scaleway.com/instance/servers) displays.
    3. Complete the following steps:
        - Choose an **Availability Zone**, which represents the geographical region where your Instance will be deployed.
        - **Choose a POP2-WIN** Instance type from the **Production-Optimized** range.
        - **Choose a Windows Server image** to run on your Instance.
        - **Name your Instance**, or leave the randomly-generated name in place. Optionally, you can add [tags](/compute/instances/concepts/#tags) to help you organize your Instance.

>         - **Add volumes**, which are storage spaces used by your Instances. A block volume with a default name and 5,000 IOPS is automatically provided for your system volume. You can customize this volume and attach up up to 16 local and/or block type volumes as needed.

Reviewer: ```suggestion
        - **Add volumes**, which are storage spaces used by your Instances. A block volume with a default name and 5,000 IOPS is automatically provided for your system volume. You can customize this volume and attach up to 16 local and/or block type volumes as needed.
```

          <Message type="important">

>              - Ensure that the volume containing your OS image has a minimum size of 10 GB. For a GPU OS, the recommended size is 125 GB.

Author: ```suggestion
             - Ensure that a volume with a Windows image has a minimum capacity of 25 GB.
```
